src/api/config/environments/development.rb

```diff
@@ -8,8 +8,8 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Do not eager load code on boot.
-  config.eager_load = false
+  # Eager load code on boot.
+  config.eager_load = true
```

Comment:
I don't mind. We already use the ``::`` several times in the app.
However, please move this in a separate commit at least as it is basically unrelated to your other changes.